server/apiCalls/transformers/memosCount.test.js

```diff
@@ -200,8 +201,8 @@ describe('Count memos and courses as one for one course with two course offering
   })
   test('One course (two offerings) and offerings has two different memos published before course start', async () => {
     const offeringsWithMemos = [
-      offering1WithDates({ memoValues: { ladokRoundIds: ['1'] }, offeringValues: { offeringId: '1' } }),
-      offering1WithDates({ memoValues: { ladokRoundIds: ['2'] }, offeringValues: { offeringId: '2' } }),
+      offering1WithDates({ memoValues: { applicationCodes: ['1'] }, offeringValues: { offeringId: '1' } }),
```

Comment:
`offeringId`  has to be removed to be sure tests working properly or replaced

Reply:
Removed